src/utils/mod.rs

```diff
@@ -1516,3 +1517,37 @@ impl From<PcrData> for TPML_DIGEST {
         tpml_digest
     }
 }
+
+fn tpm_int_to_string(num: u32) -> String {
+    u32::from_be(num)
+        .to_ne_bytes()
```

Comment:
Sorry for being a little bit stupid when it comes to endians. But why couldn't you do ```num.to_be_bytes()```?